src/components/tile/Tile.jsx

```diff
@@ -0,0 +1,13 @@
+function Tile (props) {
```

Comment:
Hier had je ipv props ook alle elementen apart kunnen benoemen, maar dit is ook goed en lijkt makkelijker te zijn ook dus goed bedacht!